tideways/provision.sh

```diff
@@ -75,15 +76,31 @@ function install_xhgui_frontend() {
         cd xhgui
         echo " * Installing xhgui"
         sudo php install.php
-        cp -f "${DIR}/config.php" "/srv/www/default/xhgui/config/config.php"
-        echo " * Restarting MongoDB"
-        service mongod restart
     else
         echo -e " * Updating xhgui..."
         cd /srv/www/default/xhgui
         git pull --rebase origin master > /dev/null 2>&1
         noroot composer update --prefer-dist > /dev/null 2>&1
     fi
+    cp -f "${DIR}/config.php" "/srv/www/default/xhgui/config/config.php"
+    if [[ ! -d "/srv/www/default/php-profiler" ]]; then
+        echo -e " * Installing php-profiler for Xhgui"
+        for version in "7.0" "7.1" "7.2" "7.3" "7.4"
+        do
+            if [[ $(command -v php$version) ]]; then
+                apt install "php${version}-sqlite3" -y &>/dev/null
+            fi
+        done
+        cd /srv/www/default
+        mkdir ./php-profiler && cd ./php-profiler
+        echo " * Installing php-profiler"
+        composer require perftools/php-profiler > /dev/null 2>&1
+        composer require perftools/xhgui-collector > /dev/null 2>&1
+    else
+        cd /srv/www/default/php-profiler
+        composer update
```

Comment:
typically should use `composer install` rather `composer update`. on first install it would not matter if you call composer install or composer update, but if lock already exists composer install is able to finish quickly because it knows what is installed.

Reply:
I think the concern here is that once it's installed it won't update to newer versions, it'll just output in the log that the lock  file is out of date with the desired requirements, and recommend running `composer update`. Since the script runs non-interactively we'd need to handle that situation.

Otherwise we'd need to commit and maintain a composer lock file

Reply:
Also the folder name is a misunderstanding,it is just an empty folder and a git clone with 2 composer package required.
I kept the name just to help to identify the tool.
In VVV we always use composer update so it is part also of our consistency.